src/frontend/scss/_main.scss

```diff
@@ -105,6 +105,7 @@
 @import './components/templates/courses/cms/blogpost_detail';
 @import './components/templates/courses/cms/program_list';
 @import './components/templates/courses/cms/program_detail';
+@import '../js/components/LtiConsumer/styles';
```

Comment:
Just move this line to line 73 (below `@import '../js/components/CourseRunEnrollment/styles';`)